packages/account/src/providers/provider.ts

```diff
@@ -21,11 +21,11 @@ import type { Predicate } from '../predicate';
 
 import { getSdk as getOperationsSdk } from './__generated__/operations';
 import type {
-  GqlChainInfoFragmentFragment,
+  GqlChainInfoFragment,
```

Comment:
I made a previous comment which I think should be the better solution but otherwise we should prefer
```typescript
import type {
	GqlFeeParameters as FeeParameters,
	GqlPredicateParameters as PredicateParameters,
	...
```

Reply:
@maschad https://github.com/FuelLabs/fuels-ts/pull/2241/commits/372fb6fb673209e15aa2230bc334ccfa2f7e034f